helloworld.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express')
+
+var app = express()
```

Comment:
Most of the tutorials I've read names it "app," but since we're only building the server, I prefer to name it "server."